src/components/loading/loading-controller.ts

```diff
@@ -35,6 +35,11 @@ import { LoadingOptions } from './loading-options';
  * `onDidDismiss` function can be called to perform an action after the loading
  * indicator is dismissed.
  *
+ * Attempting to dismiss a loading indicator before it has been completely
+ * presented will result in errors, so make sure to enclose all code paths
```

Comment:
If one follows the style guide in the current documentation they will get uncaught exceptions at relatively unpredictable times. That is really all I'm trying to avoid here.